src/templates/python/resource.ejs

```diff
@@ -35,22 +39,23 @@ class _<%= thisType %>:
     var params = paramsForAction(action);
     var pathParamNames = params.pathParams.map(function(param) { return param.name; });
     var explicitParams = params.pathParams.concat(params.explicitNonPathParams).concat(extraParam ? [extraParam] : []);
-    var documentedParams = explicitParams.concat(params.optionParams);
+    //var documentedParams = explicitParams.concat(params.optionParams);
```

Comment:
Generally we don't like to leave code in if it's commented out, unless for some reason we think it's the right code to have but for some reason temporarily we can't have it running (in which case we'd always include a comment describing why it's disabled and when we should enable it). Is there any reason to leave this line in?